style.css

```diff
@@ -0,0 +1,492 @@
+  background-size: cover;
+}
+
+.hero h1 {
+  font-family: "COCOGOOSE", sans-serif;
+  font-weight: 1000;
+  font-size: 5rem;
+  line-height: 1.1;
+  -webkit-background-clip: text;
+  background-image: url(./assets/images/hero/hbgd6.jpg);
+  background-position: left 23%, right top;
+  background-attachment: local;
+  background-size: cover;
+  color: transparent;
+}
+
+.hero2 h1 {
+  font-family: "COCOGOOSE", sans-serif;
+  font-weight: 1000;
+  font-size: 4rem;
+  line-height: 1.1;
+  text-shadow: 0 0 25px #d3d3d3, 0 0 5px #d3d3d3;
+}
+
+.hero h2 {
+  font-weight: 900;
+}
+
+/* Tools & Applications section  */
+.toolsapp {
+  background-image: url(./assets/images/tools/infoBg.png);
+}
+
+/* course Section */
+.courseSection .images {
+  width: 100%;
+  margin: 1rem;
+}
+
+.courseSection .img1 {
+  width: 50%;
+  height: 50%;
+  position: absolute;
+  left: 0;
+  top: 0;
+}
+
+.courseSection .img2 {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  width: 90%;
+  height: 90%;
+}
+
+.courseSection hr {
+  width: 10%;
+}
+
+.lineTen {
+  width: 10%;
+  height: 0.3vh;
+}
+
+.linesmall {
+  width: 5%;
+  height: 0.3vh;
+}
+
+/* footer */
+.copyright img {
+  width: 40%;
+}
+
+.copyright p {
+  font-size: 1.7vh;
+}
+
+.wedo1 {
+  background:
+    linear-gradient(rgba(255, 79, 55, 0.5), rgba(255, 79, 55, 0.5)),
+    url(./assets/images/wedos/wedo1.jpg);
+}
+
+.wedo2 {
+  background:
+    linear-gradient(rgba(255, 79, 55, 0.5), rgba(255, 79, 55, 0.5)),
+    url(./assets/images/wedos/wedo2.jpg);
+}
+
+.wedo3 {
+  background:
+    linear-gradient(rgba(255, 79, 55, 0.5), rgba(255, 79, 55, 0.5)),
+    url(./assets/images/wedos/wedo3.jpg);
+}
+
+.wedo1,
+.wedo2,
+.wedo3 {
+  height: 40vh;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+}
+
+.toolsapp .card:hover {
+  border: 1px solid white;
+}
+
+@media screen and (min-width: 768px) {
+  .width8 {
+    width: 80%;
+  }
+
+  .width5 {
+    width: 50%;
+  }
+
+  .toolsapp .card {
+    height: 45vh;
+  }
+
+  .toolsapp .card-text {
+    font-size: 2.8vh;
+  }
+
+  /* header */
+  .navbar-toggler {
+    display: none;
+  }
+
+  .desk-ctr {
+    display: flex;
+    height: 3.7rem;
+    align-items: center;
+  }
+
+  .first-nav {
+    height: 1.6rem;
+  }
+
+  .nav-link {
+    margin-left: 2rem;
+  }
+
+  .nav-item .btn {
+    border: 0.2rem solid #ec5242;
+    border-radius: 0;
+    -webkit-border-radius: 0;
+    -moz-border-radius: 0;
+    -ms-border-radius: 0;
+    -o-border-radius: 0;
+  }
+
+  /* hero section */
+  .hero,
+  .hero2 {
+    height: 100vh;
+  }
+
+  .hero2 .notep {
+    display: none;
+  }
+
+  .heroabout {
+    width: 75%;
+  }
+
+  .hero-cont {
+    padding-top: 5.6rem;
+  }
+
+  .career p {
+    width: 65%;
+  }
+
+  .hero-cont p:first-child {
+    font-size: 2.5rem;
+  }
+
+  /* Tools & Applications section  */
+  .toolsapp .btn {
+    display: none;
+  }
+
+  /* course Section */
+  .courseSection .btn {
+    display: none;
+  }
+
```

Comment:
- [ ] As the previous reviewer mention, kindly add an underline on the Partners title

![image](https://user-images.githubusercontent.com/93630700/210818122-98b27704-28ce-4f3d-8938-9df12ff6a4f4.png)